### PR TITLE
Removes the video ID field

### LINKED ISSUE #3877: remove field_video_id from image content type
image

### PR DESCRIPTION
**Pull request by itsjoekent**
Fixes #3877 

@aaronschachter

### CODE REVIEW

File: lib/modules/dosomething/dosomething_image/dosomething_image.install [hunk 1]
```diff
@@ -18,3 +18,10 @@ function dosomething_image_schema() {
 function dosomething_image_update_7001(&$sandbox) {
   field_delete_field('field_video_poster');
 }
+
+/**
+ * Deletes field_video_id.
+ */
+function dosomething_image_update_7002(&$sandbox) {
+  field_delete_field('field_video_id');
```
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_image/dosomething_image.install]: This is going to delete the field entirely from the DB, you'll see its used in other spots. You need to just delete the instance per https://github.com/DoSomething/dosomething/issues/3877#issuecomment-72870165

Comment (itsjoekent) [inline lib/modules/dosomething/dosomething_image/dosomething_image.install]: wooops

**Comment by itsjoekent** [comment]
@aaronschachter fixed da

File: lib/modules/dosomething/dosomething_image/dosomething_image.install [hunk 2]
```diff
@@ -18,3 +18,10 @@ function dosomething_image_schema() {
 function dosomething_image_update_7001(&$sandbox) {
   field_delete_field('field_video_poster');
 }
+
+/**
+ * Deletes field_video_id.
+ */
+function dosomething_image_update_7002(&$sandbox) {
+  field_delete_instance('field_video_id');
```
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_image/dosomething_image.install]: This doesn't work!  https://api.drupal.org/comment/52613#comment-52613

Please pull down new stage DB and run `drush updb` then `drush -y fr dosomething_image` to test after fixing.

**Comment by itsjoekent** [comment]
@aaronschachter updated

**Comment by aaronschachter** [comment]
If tested on a clean stage DB and verified the field no longer exists on Image node, looks good.

**Pull request merged**